locales/en/translation.json

```diff
@@ -0,0 +1,8 @@
+{
+  "Welcome to My House": "Welcome to My House",
+  "Translated content with a custom component: <2>translated link text</2>": "Translated content with a custom component: <2>translated link text</2>",
+  "Welcome to React": "",
+  "{{author}} is the author of {{book}}": "",
+  "You have {{count}} messages.": "You have {{count}} messages.",
+  "You have {{count}} messages._plural": "You have {{count}} messages."
```

Comment:
How does plural work with this library? In this example, they're both "messages". Should the first be "message"?

Reply:
Yeah. The tool that generates the json files uses the value of the <Trans> component as the default translation. I forgot to do it, but you would update the 'translation' here to handle it appropriately. Updated in new commit.

Reply:
@bhainesva Got it. How do we tell it that it should use the singular version when `count == 1` and the plural version in all other cases? Especially if we have more than one variable in the string, like `{{ count }} stores near {{ query }}`

Reply:
@JRonkin `count` is a special variable specifically for use with plurals, and has to be passed as a prop to the `Trans` component. For other interpolations you don't need a prop and can use objects as described here: https://react.i18next.com/latest/trans-component#interpolation.